apps/web/src/pages/TalentNominations/NominateTalent/components/Nominee.tsx

```diff
@@ -64,6 +207,41 @@ const Nominee = ({ nomineeQuery }: NomineeProps) => {
           description: "Subtitle for nomination nominee step",
         })}
       </p>
+      <div
+        data-h2-display="base(flex)"
+        data-h2-flex-direction="base(column)"
+        data-h2-gap="base(x1)"
+      >
+        <EmployeeSearchInput
+          id="nominee"
+          name="nominee"
+          label={intl.formatMessage({
+            defaultMessage: "Nominee's work email",
+            id: "ACCjnC",
+            description: "Label for a nominations nominee",
+          })}
+          rules={{ required: intl.formatMessage(errorMessages.required) }}
+          employeeOption={fragmentToEmployee(talentNomination.nominee)}
+          errorMessages={{
+            NO_PROFILE: {
+              title: intl.formatMessage({
+                defaultMessage: "We couldn’t find a matching candidate profile",
+                id: "fx2Y2U",
+                description:
+                  "Title of the error message when the profile of a nominee cannot be found",
+              }),
+              body: intl.formatMessage({
+                defaultMessage:
+                  "It appears that this work email address isn't linked to a profile. We require nominees to have a GC Digital Talent account in order to be nominated. Please contact the nominee and ask them to create a profile before continuing. You can save this nomination as a draft using the “Save draft” link at the end of the form.",
+                id: "KtZ5yY",
+                description:
+                  "Body of the error message when the profile of a nominee cannot be found",
```

Comment:
Clicking "Save draft" in this state appears to do nothing
So does "Next step" but that is addressed in https://github.com/GCTC-NTGC/gc-digital-talent/issues/13018

![image](https://github.com/user-attachments/assets/94eb14b4-206b-49e7-a142-8684c5660017)

Reply:
Uhhh, I wouldn't expect it to since you have an error :sweat_smile:

Reply:
The copy strongly implies that "Save draft" should still work

>Please contact the nominee and ask them to create a profile before continuing. You can save this nomination as a draft using the “Save draft” link at the end of the form.

I read that as, I must contact the nominee in order to continue and the instructions say to click "Save draft" to end my session for now and to come back later once the nominee exists

Reply:
Hmm, you are right it does but unfortunately you cannot save the form when there is an error :sweat: Not sure what we can do there? :thinking:

Reply:
Probably just a copy adjustment to be honest
But I will let @NienkeBr decide if this is confusing enough to merit adjustment or it is fine and I'm nitpicking

Reply:
Okay, looks like it should trigger a save.  So I guess we should just remove the last sentence?

Reply:
Decision in standup: OK to remove the last line about saving the nomination